heat/core/tests/test_operations.py

```diff
@@ -2,9 +2,28 @@
 import torch
 
 import heat as ht
+import numpy as np
```

Comment:
numpy should go with the third-party packages up top before importing heat

Reply:
numpy was only used for some internal tests, it isn't needed anymore, so i removed it from there